wrangler.toml

```diff
@@ -8,6 +8,9 @@ services = [
   { binding = "daadmin", service = "da-admin" }
 ]
 
+[vars]
```

Comment:
I think it can be useful to also report the stack traces on stage.

Reply:
i thought so, too. but currently, stage is also accessible publicly, so I would not add it by default.